starboard/xb1/tools/xb1_launcher.py

```diff
@@ -470,6 +474,10 @@ def Run(self):
       self._LogLn('Skipping running step.')
       return 0
 
+    status, _ = self.Run2()
+    return status == ReturnCodeStatus.CRASH
```

Comment:
This function is supposed to return an int. Can you change this so it's clear what is getting returned here?

E.g. `return 1 if status == ReturnCodeStatus.CRASH else 0`

I realize that python automatically converts booleans to integers but not everyone on the Cobalt team is a Python guru.

Reply:
Replaced with `if`.